mobile/src/main/java/org/hawkular/client/android/activity/MainActivity.java

```diff
@@ -294,6 +300,21 @@ void openDrawerMenu() {
         drawer.openDrawer(GravityCompat.START);
     }
 
+    // --- OnCLick Event for clicking Fab --- Adds a new trigger ---------
+
+    @OnClick(R.id.fab)
+    void onClickFab(){
+
+        Bundle bundle = new Bundle();
+        bundle.putString("state", "From Home");
+        AlertSetupFragment alertSetupFragment = new AlertSetupFragment();
+        alertSetupFragment.setArguments(bundle);
+
+        adapter.addFragment(alertSetupFragment,"Setup Alert");
```

Comment:
As you can see in the screenshot uploaded above the setup trigger is a fragment inside a tabbed viewpager. Actually, the current main activity layout has support for only tabs with viewpager so I had to add a new tab to setup trigger. This is temporary since I and @garg-anuj are still not sure about the placement of the fragment.